### DISCUSSION TITLE
Ane for Google Play Billing Library Version 3 ?

### DISCUSSION THREAD

**Discussion by hotdes** [airsdk/Adobe-Runtime-Support#1550] [Q&A]
Google Play currently requires Google Play Billing Library version 3.
ANE I bought 5 years ago uses Google Play Billing Library version 2.
What to do?
Are there such ANE for Google Play Billing Library version 3?

**Comment by FliplineStudios** [comment]
Distriqt's In-App Billing ANE uses Version 3, you can read more about the ANE here:
https://airnativeextensions.com/extension/com.distriqt.InAppBilling

There may be some others as well, though I've mainly stuck with the Distriqt ANEs after migrating away from the old MilkManGames ones.

**Reply by hotdes** [reply]
Ok, thanks!
